dev-python/mutagen/mutagen-1.42.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Python library to read and write audio tags for many formats"
+DESCRIPTION="Mutagen is a Python module to handle audio metadata. It supports \
+	ASF, FLAC, MP4, Monkey's Audio, MP3, Musepack, Ogg Opus, Ogg FLAC, Ogg Speex, \
+	Ogg Theora, Ogg Vorbis, True Audio, WavPack, OptimFROG, and AIFF audio files. \
+	All versions of ID3v2 are supported, and all standard ID3v2.4 frames are parsed. \
+	It can read Xing headers to accurately calculate the bitrate and length of MP3s. \
+	ID3 and APEv2 tags can be edited regardless of audio format. It can also manipulate \
+	Ogg streams on an individual packet/page level."
+HOMEPAGE="https://github.com/quodlibet/mutagen"
+COPYRIGHT="Joe Wreschnig, Michael Urman, Lukáš Lalinský, Christoph Reiter, Ben Ockmore & others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/source/m/$portName/$portName-$portVersion.tar.gz"
+CHECKSUM_SHA256="bb61e2456f59a9a4a259fbc08def6d01ba45a42da8eeaa97d00633b0ec5de71c"
+SOURCE_DIR="mutagen-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python python36)
+PYTHON_VERSIONS=(2.7 3.6)
```

Comment:
3.7 should be added too.

Reply:
Ok, I was already wondering about this. What is the actual longer term plan here? When playing with this I had some trouble since some things (e.g. pip3) use python 3.7, which is in a package called python3. Will there be only a single python3 package in the future, or will we get python36, python37, python38 etc.?

Reply:
We don't know, yet, see https://github.com/haikuports/haikuports/issues/3822
If you know more about Python and the possible compatibility between versions, help is welcome

Reply:
Good question, not sure. A new Python minor version in this case definitely requires a rebuild of all packages, but that probably is less of a mess then having packages being build against 5 different versions of Python 3.

Anyway, for now I did the changes to this package. Does this look correct:

```
PYTHON_PACKAGES=(python python36 python3)
PYTHON_VERSIONS=(2.7 3.6 3.7)
```